include/libopencm3/sam/rstc.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2015 Owen Kirby <oskirby@gmail.com>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef SAM3X_RSTC_H
+#define SAM3X_RSTC_H
+
+#include <libopencm3/cm3/common.h>
+#include <libopencm3/sam/memorymap.h>
+
+
+/* --- RSTC registers ---------------------------------------------------- */
+
+#define RSTC_CR				MMIO32(RSTC_BASE + 0x00)
+#define RSTC_SR				MMIO32(RSTC_BASE + 0x04)
+#define RSTC_MR				MMIO32(RSTC_BASE + 0x08)
+
+/* --- RSTC_CR values ----------------------------------------------------- */
```

Comment:
same here in this file.  The TC file has done it much better, you can see what the regs are.